src/course-outline/CourseOutline.jsx

```diff
@@ -89,6 +90,7 @@ const CourseOutline = ({ courseId }) => {
   } = useCourseOutline({ courseId });
 
   const [sections, setSections] = useState(sectionsList);
+  const [isSubsectionConfigure, setIsSubsectionConfigure] = useState(false);
```

Comment:
This should not be required, as `currentItem` state is set to current item i.e. section or subsection on menu button click. You can get the type i.e. category of current item from `category` field.

Reply:
@steff456 This is not resolved. We don't need this state, you can directly check inside configure modal using `category` field of currentItem state.